pallets/passkey/Cargo.toml

```diff
@@ -25,6 +25,8 @@ p256 = { workspace = true }
 base64-url = { workspace = true }
 serde_json = { workspace = true, features = ["alloc"] }
 pallet-transaction-payment = { workspace = true }
+# it does not build independently without pallet-sudo/std
+pallet-sudo = { workspace = true, optional = true }
```

Comment:
I think we can fix this from upper level but let's keep it until we figure it out